<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>864</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QProgressBar {
	border: 0.1px solid grey;
	background-color:rgb(74,75,83);
}
QProgressBar::chunk {
	background-color:rgb(0,185,84);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelbg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>864</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/bg3.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>237</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>主机温度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>237</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>主轴转速</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>237</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>副轴转速</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>240</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>主机时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>327</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>1号仓压力</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>414</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>2号仓压力</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>504</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>3号仓压力</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>593</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>4号仓压力</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>681</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>5号仓压力</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>770</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>1号工作缸油量</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>770</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>2号工作缸油量</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>770</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>3号工作缸油量</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>350</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>机械臂压力</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_wendu">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>269</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>000</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMainCircle">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>269</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>0000</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDMainCircle">
    <property name="geometry">
     <rect>
      <x>703</x>
      <y>268</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>0000</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStorePa1">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>328</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>0000 Pa</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStorePa2">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>413</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>0000 Pa</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStorePa3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>502</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>0000 Pa</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStorePa4">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>592</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>0000 Pa</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStorePa5">
    <property name="geometry">
     <rect>
      <x>134</x>
      <y>680</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>0000 Pa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_oilNum1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>800</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>000 L</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_oilNum2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>800</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>000 L</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_oilNum3">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>800</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>000 L</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelArmPress">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>350</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>0000 Pa</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>269</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>2024-1-1 0:0 0</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_cang1">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>356</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>500</number>
    </property>
    <property name="maximum">
     <number>1200</number>
    </property>
    <property name="value">
     <number>800</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_cang2">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>444</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>500</number>
    </property>
    <property name="maximum">
     <number>1200</number>
    </property>
    <property name="value">
     <number>800</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_cang3">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>532</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>500</number>
    </property>
    <property name="maximum">
     <number>1200</number>
    </property>
    <property name="value">
     <number>800</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_cang4">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>621</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>500</number>
    </property>
    <property name="maximum">
     <number>1200</number>
    </property>
    <property name="value">
     <number>800</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_cang5">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>710</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>500</number>
    </property>
    <property name="maximum">
     <number>1200</number>
    </property>
    <property name="value">
     <number>800</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_arm">
    <property name="geometry">
     <rect>
      <x>965</x>
      <y>381</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>500</number>
    </property>
    <property name="maximum">
     <number>1200</number>
    </property>
    <property name="value">
     <number>800</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
